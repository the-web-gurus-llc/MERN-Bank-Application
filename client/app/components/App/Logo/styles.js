import { headerHeight } from 'components/App/Header/styles';
import styled from 'styled-components';

export const StyledLogo = styled.div`
  height: ${headerHeight};
  color: black;
  display: flex;
  align-items: center;
  justify-content: center;
  border-bottom: 1px solid rgba(0, 0, 0, 0.12);
`;
