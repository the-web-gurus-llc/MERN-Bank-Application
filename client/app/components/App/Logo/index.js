/**
 *
 * Sidebar
 *
 */

import React from 'react';
import logo from 'images/logo.png';
import { StyledLogo } from './styles';

export default function Logo() {
  return (
    <StyledLogo>
      <img width="150" height="auto" src={logo} alt="Bank Application" />
    </StyledLogo>
  );
}
