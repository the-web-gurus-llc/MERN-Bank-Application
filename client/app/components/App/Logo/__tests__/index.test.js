import React from 'react';
import { render } from '@testing-library/react';
import Logo from '../index';

describe('<Logo />', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Logo />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
