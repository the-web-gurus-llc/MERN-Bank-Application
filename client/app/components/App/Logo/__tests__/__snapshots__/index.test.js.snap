// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should match the snapshot 1`] = `
<div
  class="Logostyle__StyledLogo-w0vuv7-0 evhvaf"
>
  <img
    alt="Bank Application"
    height="auto"
    src="IMAGE_MOCK"
    width="150"
  />
</div>
`;
